transcriptic/cli.py

```diff
@@ -410,22 +417,32 @@ def runs(ctx, project_name):
                              r['completed_at'].split("T")[0] if r['completed_at']
                              else r['created_at'].split("T")[0],
                              r['status'].replace("_", " ")])
-
-        click.echo(
-            '\n{:^120}'.format("Runs in Project '%s':\n" %
-                               get_project_name(project_name))
-        )
-        click.echo('{:^30}'.format("RUN TITLE") + "|" +
-                   '{:^30}'.format("RUN ID") + "|" +
-                   '{:^30}'.format("RUN DATE") + "|" +
-                   '{:^30}'.format('RUN STATUS'))
-        click.echo('{:-^120}'.format(''))
-        for run in run_list:
-            click.echo(u'{:^30}'.format(run[0]) + "|" +
-                       u'{:^30}'.format(run[1]) + "|" +
-                       u'{:^30}'.format(run[2]) + "|" +
-                       u'{:^30}'.format(run[3]))
-            click.echo(u'{:-^120}'.format(''))
+        if json_flag:
+            run_list = []
+            for r in req:
+                run_list.append({'title': r['title'] or "(Untitled)",
+                                 'id': r['id'],
+                                 'completed_at': r['completed_at'].split("T")[0]
+                                   if r['completed_at'] else None,
+                                 'created_at': r['created_at'].split("T")[0],
+                                 'status': r['status'].replace("_", " ")})
```

Comment:
Why do this replacement? JSON is intended to be processed by machines, not nice for humans to read